hydrus/itemhelpers.py

```diff
@@ -0,0 +1,157 @@
+"""Helper functions for items"""
+import json
+
+from flask import jsonify, request
+from hydra_python_core.doc_writer import HydraStatus, HydraError
+
+from hydrus.data import crud
+from hydrus.data.exceptions import (
+    ClassNotFound,
+    InstanceNotFound,
+    InstanceExists,
+    PropertyNotFound)
+from hydrus.helpers import (
+    set_response_headers,
+    finalize_response,
+    hydrafy,
+    getType,
+    validObject,
+    check_required_props,
+    send_sync_update,
+    get_link_props)
+from hydrus.utils import (
+    get_session,
+    get_api_name,
+    get_hydrus_server_url)
+from hydrus.socketio_factory import socketio
+
+
+def items_get_check_support(id_, class_type, class_path, path):
+    """Check if class_type supports GET operation"""
```

Comment:
@xadahiya Names and docstrings of these functions are misleading, we already have a method(named `checkClassOp`) to check whether a class supports a particular method or not.
What we are actually doing here is implementing these methods.